lib/game.rb

```diff
@@ -64,8 +64,10 @@ def select_menu(menu_option)
         gym_menu
       when '7'
         store_menu
-      when '?'
+      when 'help'
```

Comment:
It never hits the help_menu when you type `?`? Does it if you type out `help`? I wanted to keep this each to a single character. Maybe we can just do `h`?